src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +16,9 @@ public class UserService {
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
-        return null;
+        return Arrays.stream(users)
+                .filter(u -> u.getEmail().equals(email))
```

Comment:
```suggestion
                .filter(user -> user.getEmail().equals(email))
```
avoid one-letter naming even in stream chains